changelogs/fragments/20240721-azure_manage_postgresql.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - Removing tier, capacity, and size from SKU parameters from README file, both from Role Variables and Example Playbook
+  - Removing azure_manage_postgresql_postgresql_sku.tier from defalts
+  - Removing sku.tier from assert testing
```

Comment:
I don't think these description needed in the release notes, this is implicit from the first one

Reply:
removed